drivers/ukbus/pci/arch/arm64/pci_bus.c

```diff
@@ -91,6 +92,12 @@ static int arch_pci_driver_add_device(struct pci_driver *drv,
 	memcpy(&dev->addr, addr,  sizeof(dev->addr));
 	dev->drv = drv;
 
+#if CONFIG_PAGING
+	base = uk_bus_pf_devmap(base, __PAGE_SIZE);
+	if (unlikely(PTRISERR(base)))
```

Comment:
Can we print a message?
Also why `__PAGE_SIZE`?

Reply:
> Can we print a message?

Done

> Also why __PAGE_SIZE?

Because rewriting this anyway, and given this drivers (bad) design I think it's not worth to implement proper size querying. I'll add a TODO.